cgi/reset_password.pl

```diff
@@ -75,7 +75,7 @@
 			my $emails_ref = retrieve("$data_root/users/users_emails.sto");
 			if (not defined $emails_ref->{$id}) {
 				# not found, try with lower case email
-				$id = lc $id;
+				lc $id = lc $id;
```

Comment:
We have to process once all emails stored in users_emails.sto to lowercase them, we can't do that here.